### PR TITLE
Custom IP-subnet

### PR DESCRIPTION
**Pull request by timbo2k**
Hello

first of all thanks for your amazing script setting up openvpn-server - used that several times until now.
I added a minor feature for customizing the IP-subnet of openvpn-server and its clients including the regarding questions.
Successfully tested on CentOS7.

I'm open for discussion. If those changes are not wanted please just reject PR.

thanks timbo

### CODE REVIEW

**Comment by timbo2k** [comment]
just noticed that there is a similar PR (https://github.com/angristan/openvpn-install/pull/550) just a few hours ago with the same topic, did saw that before - Sorry

File: openvpn-install.sh [hunk 1]
```diff
@@ -104,8 +104,8 @@ function installUnbound () {
 			apt-get install -y unbound
 
 			# Configuration
-			echo 'interface: 10.8.0.1
-access-control: 10.8.0.1/24 allow
+			echo 'interface: $VPN_SUBNET.1
+access-control: $VPN_SUBNET.1/24 allow
```
Comment (HenryNe) [inline openvpn-install.sh]: Expressions don't expand in single quotes, use double quotes for that.

**Review by HenryNe — CHANGES_REQUESTED** [review]
I suggest to check the changes with https://www.shellcheck.net/ online.

File: openvpn-install.sh [hunk 2]
```diff
@@ -104,8 +104,8 @@ function installUnbound () {
 			apt-get install -y unbound
 
 			# Configuration
-			echo 'interface: 10.8.0.1
-access-control: 10.8.0.1/24 allow
+			echo 'interface: $VPN_SUBNET.1
```
Comment (HenryNe) [inline openvpn-install.sh:107]: Expressions don't expand in single quotes, use double quotes for that.

```suggestion
			echo "interface: $VPN_SUBNET.1
```

**Review by HenryNe — APPROVED** [review]

**Review by HenryNe — CHANGES_REQUESTED** [review]
Expressions don't expand in single quotes, and a variable in left side.
I suggest to check the changes with https://www.shellcheck.net/ online.
_Sorry for double post. I'm fighting with github._

**Comment by angristan** [comment]
Hi @timbo2k, thanks for you contribution. There is indeed a PR at #550, so we'll good with that one.

Cheers!

**Pull request closed without merging**